src/js/content/frame/index.js

```diff
@@ -50,12 +50,44 @@ const Popup = () => {
     }
     let theme = 'light';
     let colorSet = 'normal';
+    let autoAllow = false;
+    let autoAllowTimeout = 0;
+
     if (storage.preferences && storage.preferences.darkMode) {
       theme = 'dark';
     }
     if (storage.preferences.colorset) {
       colorSet = storage.preferences.colorset;
     }
+    if (storage.preferences.autoAllow) {
+      autoAllow = true;
+      autoAllowTimeout = storage.preferences.autoAllowTimeout || 10;
+    }
+
+    let continueButton;
+
+    if (autoAllow) {
+      const seconds = autoAllowTimeout;
+      setTimeout(() => onContinue, seconds * 1000);
```

Comment:
This wont call `onContinue`, it will just return it from your function.

Should be 
```setTimeout(onContinue, seconds * 1000);```